### DISCUSSION TITLE
Playing radio station does not show artist and/or title

### DISCUSSION THREAD

**Discussion by KruseCarsten** [ampache/ampache#2695] [Q&A]
Ampache version: 4.2.5 (docker ampache/ampache:latest)
Web server + version: nginx/1.18.0 (docker linuxserver/swag:latest)
Server operating system: Fedora 33 Server
Client operating system: Win10 + Chrome + Web-Player

Is there a setting at ampache.cfg.php or at station configuration to enable the web-player show artist and track title and not only show the station name?

![image](https://user-images.githubusercontent.com/74878006/104435888-34d42480-558d-11eb-9187-524390232bc6.png)

**Comment by lachlan-00** [comment]
It could be read from the stream but i think because it's not an Ampache object it's been set up as a generic object.

It's definitely not impossible for it to be done though.

**Comment by usox** [comment]
Maybe, I'll look into it some day - need some diversion. And if this helps someone who listens to Radio Bob like me,  it's a win-win :)

**Reply by SePHTaN** [reply]
@usox 
[This might be useful](https://stackoverflow.com/questions/4911062/pulling-track-info-from-an-audio-stream-using-php/4914538#4914538)
[Also this](https://people.kth.se/~johanmon/dse/casty.pdf)
I have at the moment not enough insight in ampache to give this a try, wasn't even able to quickly find which part is responsible for starting the play of radiostations at all.

**Reply by usox** [reply]
@SePHTaN Thanks, I'm quite familiar with those headers. Iirc the way, ampache handles the stream was the problem in here.
